### PR TITLE
finish 'ListOfCryptocurrency' card

### PR DESCRIPTION
**Pull request by salvaarmanasco**
finish the first Card!

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,118 +1,31 @@
...
-function App(): JSX.Element {
-  const isDarkMode = useColorScheme() === 'dark';
-
-  const backgroundStyle = {
-    backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
-  };
-
+// -----------------------------------------Styled Components-------------------------------------------//
+import {ThemeProvider} from 'styled-components/native';
+import THEME from './src/styles/theme';
```
Comment (tomasdematos) [inline App.tsx:4]: renate to theme

File: App.tsx [hunk 2]
```diff
@@ -1,118 +1,31 @@
...
-          </Section>
-          <LearnMoreLinks />
-        </View>
-      </ScrollView>
-    </SafeAreaView>
+          <Stack.Screen name="Home" component={Home} />
+        </Stack.Navigator>
+      </NavigationContainer>
+    </ThemeProvider>
+    // </Provider>
```
Comment (tomasdematos) [inline App.tsx:27]: delete this comment

File: src/screens/home/Home.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import { ButtonAdd, ButtonText } from "./components/styles";
+
+interface Navigation {
+  navigation: {
+    navigate: (routeName: string) => void;
+  };
+}
+
+const Home = ({ navigation }: Navigation) => {
+  return (
```
Comment (tomasdematos) [inline src/screens/home/Home.tsx:13]: use implicit return:

const Home = ({ navigation }: Navigation) => (<>.....</>)

File: src/screens/home/components/AddButton.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { View, Text } from "react-native";
+import React from "react";
+
+import { ButtonAdd, ButtonText } from "./styles";
+
+const AddButton = () => {
```
Comment (tomasdematos) [inline src/screens/home/components/AddButton.tsx:6]: use implicit return

File: src/screens/home/components/Coin.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Text, Image } from "react-native";
+import CoinsInfo from "types/CoinsInfo";
+
+const arrowUp = require("../../../../assets/img/arrowup.png");
```
Comment (tomasdematos) [inline src/screens/home/components/Coin.tsx:5]: use import in this two cases

File: src/screens/home/components/Coin.tsx [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+  ContainerNameAndSymbol,
+  ContainerPriceAndPercent,
+  Name,
+  Symbol,
+  PriceText,
+  Percent,
+  CryptoText,
+  Arrows,
+} from "./styles";
+const Coin = ({ item }: { item: CoinsInfo }) => {
```
Comment (tomasdematos) [inline src/screens/home/components/Coin.tsx:20]: add a space above

File: src/screens/home/components/CryptoList.tsx [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+    id: "3",
+    name: "XRP",
+    symbol: "XRP",
+    price: 0.3928,
+    percent: -2.76,
+    img: XRP,
+  },
+];
+
+console.log(CryptoArr);
```
Comment (tomasdematos) [inline src/screens/home/components/CryptoList.tsx:38]: delete this log

File: src/screens/home/components/styles.tsx [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+  flex: 1;
+`;
+export const ContainerPriceAndPercent = styled.View`
+  flex-direction: column;
+  align-items: flex-end;
+  flex-wrap: wrap;
+  justify-self: end;
+`;
+
+export const ImageContainer = styled.View``;
```
Comment (tomasdematos) [inline src/screens/home/components/styles.tsx:49]: empty style

File: src/screens/home/components/Coin/index.tsx [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+  PriceText,
+  Percent,
+  Arrows,
+  PercentUp,
+  PercentDown,
+  ArrowContainer,
+} from './styles';
+
+const Coin = ({item}: {item: CoinsInfo}) => {
+  return (
```
Comment (tomasdematos) [inline src/screens/home/components/Coin/index.tsx:23]: Check the implict return in all the files, 

if you don't have anything before the return use implicit, like in this case

File: src/screens/home/styles.ts [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+export const ButtonAdd = styled.TouchableOpacity`
+  display: flex;
+  align-items: center;
+  font-size: 30px;
+  padding-top: 40px;
+`;
+
+export const ButtonText = styled.Text`
+  font-size: 16px;
+  font-weight: 400;
```
Comment (tomasdematos) [inline src/screens/home/styles.ts:12]: font 400 is already the default weight

File: App.tsx [hunk 11]
```diff
@@ -1,118 +1,30 @@
...
+import {NavigationContainer} from '@react-navigation/native';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+// -----------------------------------------Components-------------------------------------------//
+import Home from './src/screens/home/Home';
+// -----------------------------------------Redux-------------------------------------------//
+
+const Stack = createNativeStackNavigator();
+
+const App = () => {
   return (
```
Comment (tomasdematos) [inline App.tsx:15]: use impllicit

Comment (salvaarmanasco) [reply App.tsx]: corregido!

**Pull request merged**